website/docs/docs/cloud/about-cloud/regions-ip-addresses.md

```diff
@@ -18,3 +18,13 @@ dbt Cloud is [hosted](/docs/cloud/about-cloud/architecture) in multiple regions
 
 
 [^1]: These regions support [multi-tenant](/docs/cloud/about-cloud/tenancy) deployment environments hosted by dbt Labs.
+
+## Static IP addresses
+
+dbt Cloud, like many cloud services, relies on underlying AWS cloud infrastructure for operations. Due to this, while we can offer exact URLs for access, we're unable to provide a list of static IP addresses to configure connections.
+
+1. Dynamic IP Addresses &mdash; Our cloud infrastructure is built upon Amazon Web Services (AWS). Due to the adaptive nature of cloud services, while dbt Cloud can provide exact URLs for streamlined access, the underlying IP addresses are dynamic and will change occasionally. AWS manages the IP ranges and may change them according to their operational requirements.
```

Comment:
```suggestion
1. Dynamic IP addresses &mdash; Our cloud infrastructure uses Amazon Web Services (AWS). dbt Cloud offers exact URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses change occasionally. This is because AWS manages the IP ranges and may change them according to their operational needs.
```